test/services/credential-service/credentials.test.js

```diff
@@ -251,27 +250,6 @@ describe('Credential service tests', () => {
         .be.rejectedWith('one or more scopes don\'t exist');
     });
 
-    it('should use default property if not defined', () => {
```

Comment:
why it is removed ?

Reply:
I can put this back if you want, but basically this is testing something that now is handled by JSON Schema library (ajv) — and it's guaranteed to work. Previously it made sense, because it was handled with custom code, but now I think this can be safely removed.

In particular, default values are automatically filled by ajv thanks to the JSON Schema definition. So this test will pass, always.

Reply:
same as above, no doubt it will work, but what if I remove ajv itself ? how do I know it was removed